cmd/traffic/cmd/manager/manager.go

```diff
@@ -152,6 +195,21 @@ func (s *service) servePrometheus(ctx context.Context) error {
 		return int(atomic.LoadInt32(&s.activeGrpcRequests))
 	})
 
+	labels := []string{"client", "install_id"}
+	InterceptCounter = newCounterVecFunc("intercept_count", "The total number of global intercepts by user", labels)
+	InterceptActiveStatusGauge = newGaugeVecFunc("intercept_active_status",
+		"Flag to indicate when an intercept is active. 1 for active, 0 for not active.", append(labels, "workload"))
+	ConnectCounter = newCounterVecFunc("connect_count", "The total number of connects by user", labels)
+	ConnectActiveStatusGauge = newGaugeVecFunc("connect_active_status", "Flag to indicate when a connect is active. 1 for active, 0 for not active.", labels)
+
+	s.state.SetConnectionEstablishedFinalizer(func(client, installId string) {
+		SetGauge(ConnectActiveStatusGauge, client, installId, nil, 0)
+	})
+
+	s.state.SetInterceptEstablishedFinalizer(func(client, installId string, workload *string) {
```

Comment:
and this one `SetAllInterceptsFinalizer` and also get a `*rpc.ClientInfo` rather than a name + install-id.